#include "ScalarConverter.hpp"

int main()
{
    try
    { 


    }
    catch(const std::exception &e)
    {
        std::cerr << "\e[1;31mException caught: \e[0m" << e.what() << std::endl;
    }

    std::cout << std::endl;

    return 0;
}
